packages/notebook-extension/src/celltaglist.tsx

```diff
@@ -0,0 +1,165 @@
+/*
+ * Copyright (c) Jupyter Development Team.
+ * Distributed under the terms of the Modified BSD License.
+ */
+
+import { NotebookPanel } from '@jupyterlab/notebook';
+import React from 'react';
+//import PropTypes from 'prop-types';
+import { ReactWidget, VDomModel, VDomRenderer } from '@jupyterlab/apputils';
+import { CommandRegistry } from '@lumino/commands';
+
+interface IPropsFilterButton {
+  onClick: () => void;
+  sectionLabel: string;
+  buttonLabel: string;
+}
+
+export function FilterButtonComponent(props: IPropsFilterButton) {
+  return (
+    <div>
+      <p> {props.sectionLabel}</p>
+      <button type="button" onClick={props.onClick}>
+        {props.buttonLabel}
+      </button>
+    </div>
+  );
+}
+
+export class FilterButtonWidget extends ReactWidget {
+  private _commands: CommandRegistry;
+  private _commandID: string;
+  private _buttonLabel: string;
+  private _sectionLabel: string;
+  constructor(
+    commands: CommandRegistry,
+    commandID: string,
+    sectionLabel: string,
```

Comment:
```suggestion
```

`sectionLabel` appears to be set to `''` in the only usage I see, should it be removed?